metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -1181,18 +1181,16 @@ def gen_schema_key(self, db_name: str, schema: str) -> PlatformKey:
             project_id=db_name,
             dataset_id=schema,
             platform=self.platform,
-            instance=self.config.platform_instance
-            if self.config.platform_instance is not None
-            else self.config.env,
+            instance=self.config.platform_instance,
+            backcompat_instance_for_guid=self.config.env,
         )
 
     def gen_database_key(self, database: str) -> PlatformKey:
         return ProjectIdKey(
             project_id=database,
             platform=self.platform,
-            instance=self.config.platform_instance
-            if self.config.platform_instance is not None
-            else self.config.env,
+            instance=self.config.platform_instance,
```

Comment:
Same here. Platform instance not supported for bigquery, so not sure if make sense to add instance for `ProjectIdKey`.